### PR TITLE
Feature/mvp add events exop personalisation apis

### PR DESCRIPTION
**Pull request by WillTaylor22**
- [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.

### CODE REVIEW

File: directory_api_client/personalisation.py [hunk 1]
```diff
@@ -3,10 +3,24 @@
...
 
 
 class PersonalisationAPIClient(AbstractAPIClient):
     authenticator = SessionSSOAuthenticator
 
     def user_location_create(self, sso_session_id, data):
         return self.post(url=url_user_location_create, data=data, authenticator=self.authenticator(sso_session_id))
+
+    def events_by_location_list(self, sso_session_id, lat='', lng=''):
+        return self.get(url=url_events, params={
```
Comment (richtier) [inline directory_api_client/personalisation.py]: as this is authenticating via sso_session_id the upstream view can determine the sso_id via `request.sso.id` so we could remove `sso_id` from `params`

File: directory_api_client/personalisation.py [hunk 2]
```diff
@@ -3,10 +3,24 @@
...
+          'sso_id': sso_session_id,
+          'lat': lat,
+          'lng': lng
+        }, authenticator=self.authenticator(sso_session_id))
+
+    def export_opportunities_by_relevance_list(self, sso_session_id):
+        return self.get(url=url_export_opportunities, params={
+          'sso_id': sso_session_id
```
Comment (richtier) [inline directory_api_client/personalisation.py]: similar to above, canm remove `sso_id` from params as we're using the sso authentication

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## [21.3.0](https://pypi.org/project/directory-api-client/21.2.2/) (2020-02-18)
+## [21.3.0](https://pypi.org/project/directory-api-client/21.4.0/) (2020-02-20)
```
Comment (eadpearce) [inline CHANGELOG.md]: first bit of text also need to be `21.4.0`

Comment (WillTaylor22) [reply CHANGELOG.md]: Thanks! Great spot

**Pull request merged**